File: test-schemas-domain-classes/src/main/scala/testdomains/generic/PropertyNames.java
package testdomains.generic;

import java.util.HashSet;
import java.util.Set;

public class PropertyNames {


public static final String INT_LIST = "int_list";


public static final String INT_MANDATORY = "int_mandatory";


public static final String INT_OPTIONAL = "int_optional";


public static final String STRING_LIST = "string_list";


public static final String STRING_MANDATORY = "string_mandatory";


public static final String STRING_OPTIONAL = "string_optional";

public static Set<String> ALL = new HashSet<String>() {{
add(INT_LIST);
add(INT_MANDATORY);
add(INT_OPTIONAL);
add(STRING_LIST);
add(STRING_MANDATORY);
add(STRING_OPTIONAL);
}};

}

File: test-schemas-domain-classes/src/main/scala/testdomains/generic/PropertyNames.scala
package testdomains.generic

import java.util.{HashSet, Set}
import scala.jdk.CollectionConverters.SeqHasAsJava

object PropertyNames {
  val IntList: String         = "int_list"
  val IntMandatory: String    = "int_mandatory"
  val IntOptional: String     = "int_optional"
  val StringList: String      = "string_list"
  val StringMandatory: String = "string_mandatory"
  val StringOptional: String  = "string_optional"

  /** /** This is a contained node */ */
  val ContainedNodeB: String = "contained_node_b"

  val All: Set[String] =
    new HashSet[String](Seq(IntList, IntMandatory, IntOptional, StringList, StringMandatory, StringOptional, ContainedNodeB).asJava)
}
